openvdb_houdini/openvdb_houdini/SOP_OpenVDB_Clip.cc

```diff
@@ -82,23 +84,36 @@ newSopOperator(OP_OperatorTable* table)
             "If enabled, keep voxels that lie inside the clipping region,"
             " otherwise keep voxels that lie outside the clipping region."));
 
-    parms.add(hutil::ParmFactory(PRM_STRING, "clipper", "Clip To")
+    parms.add(hutil::ParmFactory(PRM_STRING | PRM_TYPE_JOIN_NEXT, "clipper", "Clip To")
         .setChoiceListItems(PRM_CHOICELIST_SINGLE, {
             "camera",   "Camera",
-            "geometry", "Geometry Bounding Box",
+            "geometry", "Geometry",
             "mask",     "Mask VDB"
         })
         .setDefault("geometry")
         .setTooltip("Specify how the clipping region should be defined.")
-        .setDocumentation("\
-How to define the clipping region\n\
-\n\
-Camera:\n\
-    Use a camera frustum as the clipping region.\n\
-Geometry:\n\
-    Use the bounding box of geometry from the second input as the clipping region.\n\
-Mask VDB:\n\
-    Use the active voxels of a VDB volume from the second input as a clipping mask.\n"));
+        .setDocumentation("How to define the clipping region\n\n"
+            "Camera:\n"
+            "    Use a camera frustum as the clipping region.\n"
+            "Geometry:\n"
+            "    Use the bounding box of geometry from the second input as the clipping region.\n"
+            "Mask VDB:\n"
+            "    Use the active voxels of a VDB volume from the second input as a clipping mask.\n"));
+
+    parms.add(hutil::ParmFactory(PRM_TOGGLE_E, "legacycamclip", "Legacy")
+        .setDefault(PRMzeroDefaults)
+        .setInvisible()
+        .setTooltip("Enables legacy camera clipping implementation and controls")
+        .setDocumentation(
+            "Enables legacy camera clipping implementation and controls.\n\n"
+            
+            "Implementation used prior to (OpenVDB version - / Houdini -). Overestimates "
```

Comment:
I've bookmarked the doc-string here so that version numbers can be added when they are known later.